bool validity			# Validity

float32 meas_vx			# Body measurement vx
float32 meas_vy			# Body measurement vy
float32 meas_vz			# Body measurement vz

float32 est_vx			# Body inertial vx
float32 est_vy			# Body inertial vy
float32 est_vz			# Body inertial vz

float32 u			# Latent scaling
